unfetter-discover/stix
# relax types to use in ctf

supports unfetter-discover/unfetter#984

# problem
some of the classes needed to relax their types
importing is not ideal at times

# solution
removed the dist directory, to allow for cleaner looking imports
relaxed some constraints on kill chain phase and attack pattern, allowing us to use this in ctf-ingest project

imports might need to be specific at the moment
```
import { AssessedObject } from 'stix/assess/v3/assessed-object';
import { Question } from 'stix/assess/v3/question';
import { AttackPattern } from 'stix/unfetter/attack-pattern';
```

# to test
* check travis build
* wait to test this in the ui and ctf in a future PR
